ush/parsing_namelists_FV3.sh

```diff
@@ -80,674 +80,419 @@ else
+local ADJUST_DRY_MASS=${adjust_dry_mass:-".true."}
+local DRY_MASS=${dry_mass:-98320.0}
+local CONSV_TE=${consv_te}
+local DO_SAT_ADJ=${do_sat_adj:-".false."}
+local PRINT_FREQ=${print_freq}
+local WARM_START=${warm_start}
+local NO_DYCORE=${no_dycore}
+local AGRID_VEL_RST=${agrid_vel_rst:-".true."}
+local READ_INCREMENT=${read_increment}
+local RES_LATLON_DYNAMICS=${res_latlon_dynamics}
+local ATM_IGNORE_RST_CKSUM=.false.
+local INCREMENT_FILE_ON_NATIVE_GRID=.false.
+
+local FILTERED_TERRAIN=${filtered_terrain}
+local NPZP=${LEVS} #levp
+local GFS_DWINDS=${gfs_dwinds}
+
+local FHZERO=${FHZERO:-6}
+local H2O_PHYS=${h2o_phys:-".true."}
+local LDIAG3D=${ldiag3d:-".false."}
+local QDIAG3D=${qdiag3d:-".false."}
+local PRINT_DIFF_PGR=${print_diff_pgr:-".false."}
+local FHCYC=${FHCYC}
+local USE_UFO=${use_ufo:-".true."}
+local PRE_RAD=${pre_rad:-".false."}
+local IMP_PHYSICS=${imp_physics:-"99"}
+
+local default_dt_inner=$(( DELTIM/2 ))
+local IOVR=${iovr:-"3"}
+local LTAEROSOL=${ltaerosol:-".false."}
+local LRADAR=${lradar:-".true."}
+local TTENDLIM=${ttendlim:-"-999"}
+local DT_INNER=${dt_inner:-"${default_dt_inner}"}
+local SEDI_SEMI=${sedi_semi:-".true."}
+local DECFL=${decfl:-"10"}
+local OZ_PHYS_NEW=${oz_phys:-".false."}
+local OZ_PHYS_OLD=${oz_phys_2015:-".true."}
+local LSOIL_LSM=${lsoil_lsm:-"4"}
+local DO_MYNNEDMF=${do_mynnedmf:-".false."}
+local DO_MYNNSFCLAY=${do_mynnsfclay:-".false."}
+local ICLOUD_BL=${icloud_bl:-"1"}
+local BL_MYNN_EDMF=${bl_mynn_edmf:-"1"}
+local BL_MYNN_TKEADVECT=${bl_mynn_tkeadvect:-".true."}
+local BL_MYNN_EDMF_MOM=${bl_mynn_edmf_mom:-"1"}
+local DO_UGWP=${do_ugwp:-".false."}
+local DO_TOFD=${do_tofd:-".false."}
+local GWD_OPT=${gwd_opt:-"2"}
+local DO_UGWP_V0=${do_ugwp_v0:-".false."}
+local DO_UGWP_V1=${do_ugwp_v1:-".true."}
+local DO_UGWP_V0_OROG_ONLY=${do_ugwp_v0_orog_only:-".false."}
+local DO_UGWP_V0_NST_ONLY=${do_ugwp_v0_nst_only:-".false."}
+local DO_GSL_DRAG_LS_BL=${do_gsl_drag_ls_bl:-".true."}
+local DO_GSL_DRAG_SS=${do_gsl_drag_ss:-".true."}
+local DO_GSL_DRAG_TOFD=${do_gsl_drag_tofd:-".true."}
+local DO_GWD_OPT_PSL=${do_gwd_opt_psl:-".false."}
+local DO_UGWP_V1_OROG_ONLY=${do_ugwp_v1_orog_only:-".false."}
+local MIN_LAKEICE=${min_lakeice:-"0.15"}
+local MIN_SEAICE=${min_seaice:-"0.15"}
+local USE_CICE_ALB=${use_cice_alb:-".false."}
+
+local PDFCLD=${pdfcld:-".false."}
+local FHSWR=${FHSWR:-"3600."}
+local FHLWR=${FHLWR:-"3600."}
+local IALB=${IALB:-"1"}
+local IEMS=${IEMS:-"1"}
+local IAER=${IAER}
+local ICLIQ_SW=${icliq_sw:-"2"}
+local ICO2=${ICO2}
+local ISUBC_SW=${isubc_sw:-"2"}
+local ISUBC_LW=${isubc_lw:-"2"}
+local ISOL=${ISOL:-"2"}
+local LWHTR=${lwhtr:-".true."}
+local SWHTR=${swhtr:-".true."}
+local CNVGWD=${cnvgwd:-".true."}
+local SHAL_CNV=${shal_cnv:-".true."}
+local CAL_PRE=${cal_pre:-".true."}
+local REDRAG=${redrag:-".true."}
+local DSPHEAT=${dspheat:-".true."}
+local HYBEDMF=${hybedmf:-".false."}
+local SATMEDMF=${satmedmf-".true."}
+local ISATMEDMF=${isatmedmf-"1"}
```

Comment:
I think this was a typo in the develop version:
```suggestion
local ISATMEDMF=${isatmedmf:-"1"}
```